app/assets/stylesheets/post.css

```diff
@@ -21,11 +21,19 @@
   flex-direction: column;
   border: 5px solid blue;
```

Comment:
- [ ] It would be better to edit the README file so that we provide all the information needed for anyone to run the app locally. For example; `bundle install`, `rails db:create`, `rails db:migrate`, `rails db:seed` if we have seeded the data and `rails s` to run the app locally

Reply:
I have some commands on my README at the moment, I just need to update it ith the `rspec` command  to update the README file. Thanks.